### PR TITLE
chore(kiboComponent) : use kiboPrice and kiboApplyCoupon

### PR DESCRIPTION
**Pull request by kibo-susantasamanta**

### CODE REVIEW

File: lib/getters/cartGetters.ts [hunk 1]
```diff
@@ -131,6 +131,13 @@ const getFulfillmentLocation = (item: CartItem, locations): string => {
   return locations.find((location) => location.code === item?.fulfillmentLocationCode)?.name
 }
 
+const getAddToCartTotalPrice = (item: CartItem): Number => {
+  const regular = item.product?.price?.price
+  const special = item.product?.price?.salePrice
+  const qty = item?.quantity ? item?.quantity : 0
```
Comment (kibo-kevinwatts) [inline lib/getters/cartGetters.ts]: don't calculate prices, totals, etc client side. 
slack message sent with more details on where to get this data

File: components/AddToCartConfirmationModal.vue [hunk 2]
```diff
@@ -152,6 +152,7 @@ export default {
     const cartItemImage = computed(() => cartGetters.getItemImage(cartItem.value))
     const cartItemOptions = computed(() => cartGetters.getItemOptions(cartItem.value))
     const cartItemPrice = computed(() => cartGetters.getItemPrice(cartItem.value))
+    const getCartItemTotalPrice = computed(() => cartGetters.getAddToCartTotalPrice(cartItem.value))
```
Comment (kibo-kevinwatts) [inline components/AddToCartConfirmationModal.vue]: the leading "get" in variable name sounds like a function, use something like cartItemTotal

**Review by kibo-kevinwatts — CHANGES_REQUESTED** [review]

**Review by kibo-sushant — APPROVED** [review]

File: lib/getters/cartGetters.ts (hunk 1 above)
Comment (kibo-susantasamanta) [reply lib/getters/cartGetters.ts]: I saw your slack message but one doubt I have. CartItem 'total' is giving amount based on total qty for the item if item already added to cart then also it's including old qty.. So, is that correct ?

File: components/AddToCartConfirmationModal.vue (hunk 2 above)
Comment (kibo-susantasamanta) [reply components/AddToCartConfirmationModal.vue]: Updated.

File: lib/getters/cartGetters.ts (hunk 1 above)
Comment (kibo-susantasamanta) [reply lib/getters/cartGetters.ts]: Modified pricing stuff is removed now, will be handled in a separate ticket.

File: components/AddToCartConfirmationModal.vue (hunk 2 above)
Comment (kibo-susantasamanta) [reply components/AddToCartConfirmationModal.vue]: This is removed , will be handled in a separate ticket.

**Review by kibo-kevinwatts — APPROVED** [review]

**Pull request merged**